src/desktop/apps/consign/client/reducers.js

```diff
@@ -53,12 +54,14 @@ const initialState = {
     location_state: '',
     location_country: '',
     medium: '',
+    minimum_price_yesno: false,
```

Comment:
I think because this field is included in this hash, it will also be sent over to convection when we try to create a submission. It's maybe fine, but if not we should make sure to scrub it out somewhere [around here](https://github.com/artsy/force/blob/master/src/desktop/apps/consign/client/actions.js#L222).

Reply:
I works fine, and is ignored by convection. I can change it if you think it's important, or not.